tempesta_fw/tls.c

```diff
@@ -284,8 +291,7 @@ tfw_tls_encrypt(struct sock *sk, struct sk_buff *skb, unsigned int limit)
 			break;
 
 		/* @next has original seqnos, so advance both of them. */
-		tcb->seq += head_sz;
-		tcb->end_seq += head_sz;
+		tfw_tls_tcp_propagate_dseq(sk, skb_tail);
```

Comment:
If the reason for the change is that we `skb_tail` can contain seqnos not updated from previous run and we have to move them further than `head_sz`, then please update the comment to say what's happening here. The seqno games are quite subtle, so it'd be good to have good comments here.

I guess now you have warning about unused `tcb` in non-debugging build. I propose just to use `TCP_SKB_CB(next)` in `T_DBG3()`.

Reply:
I've changed comment to mention why we are calling `tfw_tls_tcp_propagate_dseq()` and why `skb_tail` is used.